linux/software/libcamera/README.md

```diff
@@ -0,0 +1,75 @@
+# _libcamera_ installation for Raspberry Pi
+
+## Preparing your Pi
+
+Your Raspberry Pi should be running the latest version of the Raspbian OS (_Buster_ at the time of writing), and the camera and I2C interfaces must both be enabled (check the _Interfaces_ tab of the _Raspberry Pi Configuration_ tool, from the _Preferences_ menu). First ensure your system, firmware and all its applications and repositories are up to date by entering the following commands into a terminal window.
+
+```bash
+sudo apt update
+sudo apt full-upgrade
+```
+
+Currently (May 2020) the necessary _libcamera_ support has not yet been merged into the standard Raspbian OS release, therefore it is necessary to install the latest release candidate. To do this, first reboot your Pi, and then use
+
+```bash
+sudo rpi-update
+```
+
+**WARNING**: Note that the release candidate is not as thoroughly tested as an official release. If your Raspberry Pi contains important or critical data we would strongly advise that it is backed up first, or that a fresh SD card is used for the purpose of trying _libcamera_.
+
+Next, the `/boot/config.txt` file must be updated to load and use the camera driver, by adding the following lines to the bottom. Currently we also need to update the GPU's `core_freq_min` though this will become unnecessary in due course after further updates.
+
+```bash
+dtoverlay=imx219
+core_freq_min=250
+```
+
+If you are using a sensor other than the `imx219` you will need to supply the alternative name here (for example, `ov5647` for the V1 camera, or `imx477` for the HQ Cam for which support will be available shortly).
```

Comment:
I guess it can be inferred form the information provided, but probably best to explicitly state that imx219 is the V2 camera?